openedx/core/djangoapps/oauth_dispatch/adapters/dot.py

```diff
@@ -81,18 +93,17 @@ def get_token_scope_names(self, token):
         """
         return list(token.scopes)
 
-    def is_client_restricted(self, client_id):
+    def is_client_restricted(self, client):
         """
         Returns true if the client is set up as a RestrictedApplication.
         """
-        application = self.get_client(client_id=client_id)
-        return RestrictedApplication.objects.filter(application=application).exists()
+        return RestrictedApplication.objects.filter(application=client).exists()
 
-    def get_authorization_filters(self, client_id):
+    def get_authorization_filters(self, client):
         """
         Get the authorization filters for the given client application.
         """
-        application = self.get_client(client_id=client_id)
+        application = client
```

Comment:
Yeah - I wonder why they chose to rename it to `application` in DOT.  The RFC still calls it `client` and this adds a needless translation.  I'm not making this method signature name change since other methods on the interface also use the common "client" terminology.